packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -224,6 +250,10 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
     });
   }
 
+  void _onShowFocusHighlight(bool showHighlight) {
+    setState(() { isFocused = showHighlight; });
```

Comment:
```suggestion
    setState(() { 
      isFocused = showHighlight;
    });
```

AFAIK, it is preferred to split lines in the curly braces even if it contains a single line.